plugins/catalog-backend-module-msgraph/api-report.md

```diff
@@ -131,6 +134,20 @@ export class MicrosoftGraphOrgEntityProvider implements EntityProvider {
   read(options?: { logger?: Logger }): Promise<void>;
 }
 
+// @alpha
+export const microsoftGraphOrgEntityProviderCatalogModule: (
+  options?: MicrosoftGraphOrgEntityProviderCatalogModuleOptions | undefined,
+) => BackendFeature;
+
+// @alpha
+export interface MicrosoftGraphOrgEntityProviderCatalogModuleOptions {
+  groupTransformer?: GroupTransformer | Record<string, GroupTransformer>;
```

Comment:
I see that this just follows the existing convention, so no action for now. But I wonder if it had been useful to instead pass in the provider ID as one of the parameters to the transformer instead of making these be maps.